cmd/zpios/zpios_main.c

```diff
@@ -551,6 +551,7 @@ run_one(cmd_args_t *args, uint32_t id, uint32_t T, uint32_t N,
 
 	if (args->verbose) {
 		rc2 = read(zpiosctl_fd, zpios_buffer, zpios_buffer_size - 1);
+		zpios_buffer[zpios_buffer_size - 1] = '\0';
```

Comment:
Yes, in fact this is not a bug, but for the coverity in the main. I'm done as you suggest, thanks for review :smile: